pkg/api/helpers.go

```diff
@@ -52,3 +54,19 @@ func GetFieldLabelConversionFunc(supportedLabels map[string]string, overrideLabe
 		return "", "", fmt.Errorf("field label not supported: %s", label)
 	}
 }
+
+// TODO: Move this upstream
+func ExportObjectMeta(objMeta *kapi.ObjectMeta, exact bool) {
```

Comment:
I see you've just copied the method from `export.go` as is, still since this will be part of public API I'd make it return the modified object.